L73-java-binderchannel/wireformat.md

```diff
@@ -229,7 +253,53 @@ stream transactions until we receive an acknowledgement message from the transpo
 peer. Each transport sends an acknowledgement for each 16KB of stream transaction
 received, which should avoid blocking the transport most of the time.
 
+## Stream Flow Control
+
+While transport flow control limits consumption of scarce buffer resources at
+the Android/binder layer, stream flow control limits the amount of buffering at
+the gRPC layer. The resulting back pressure lets an application adjust its rate
+of message production to match the remote reader's rate of stream consumption.
+
+### Sender Responsibilities
+Each stream sender (both client and server) must keep track of the amount of
+space remaining in its peer's receive window, decrementing this counter by the
+`bytes data` or `parcelable` size of each `rpc transaction` it sends and
+incrementing it upon receipt of a `stream receive window update`. A stream
+transaction must not be sent unless there's room for its data in the peer's
+window. Availability of peer receive window space should be exposed to the
+sending application using the language-specific stream readiness API. 
+
+The initial receive window size of all new streams is specified by a peer in its
+`setup transport transaction` and cannot be changed for the life of a transport.
```

Comment:
In the next section it is described how the receiver can change the window size, so this should probably be rephrased to indicate that the sender can't change it.

Reply:
I rephrased to make a stronger distinction between the window size of an established stream and the initial window size for new streams. LMK if this doesn't address your comment.